### ISSUE TITLE
Meta data storage in data.frames

### ISSUE THREAD

**Issue by MansMeg** [ajschumacher/rjstat#12]
Hi!

Im working with the pxweb R package and I have a problem that we seem to share, a standard on how to append meta data to R data.frame object.

I have been thinking of using an attribute, say "metadat" with a list with the metadata object. Have you given this problem any thoughts?

**Comment by hmalmedal** [comment]
I have thought about it, but I couldn't come up with a good solution. When printing a data frame, attributes are not shown, so it is not easy to discover them.

**Comment by MansMeg** [comment]
Thats true. But as a part of the rjsonstat package I guess one could create a new class that inherits from data.frame (maybe data.frame.meta) that only has a new print method for data.frames created by json-stat. I guess that should solve the problem?

**Comment by hmalmedal** [comment]
Perhaps. I'm not very familiar with creating new classes and methods.

**Comment by MansMeg** [comment]
I'm thinking of solutions for the pxweb package so I can try to implement something later on.
